src/components/sign-up-second/index.tsx

```diff
@@ -118,6 +118,7 @@ function SignUpSecond({ role, onNext }: IProps): JSX.Element {
               <MuiTelInput
                 {...field}
                 onChange={(value: string): void => {
```

Comment:
can we use yup validation here?

Reply:
These conditions are used to prevent the user from entering 0 and 1 in the input. If I used only Yup validation, user will see error message, but he will have ability to enter this values in input